docker-for-mac/release-notes.md

```diff
@@ -13,6 +13,14 @@ This page contains information about the new features, improvements, known issue
 For information about Edge releases, see the [Edge release notes](edge-release-notes). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.2.0.4
+2020-03-
+
+### Bug fixes and minor changes
+
+- Kubernetes: Persistent volumes created by claims are now stored in the virtual machine. Fixes [docker/for-win#5665](https://github.com/docker/for-win/issues/5665).
```

Comment:
This has a for-win ticket. It applies on Mac too, does it?

Reply:
Yes, this note is also present in the Mac release notes.